### PR TITLE
Update lint settings

### PR DESCRIPTION
**Pull request by lukaspie**
In line with https://github.com/FAIRmat-NFDI/pynxtools/pull/248

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -53,12 +53,12 @@ n2n = "nyaml.cli:launch_tool"
 
 [tool.ruff]
 include = ["pynxtools/*.py", "tests/*.py"]
```
Comment (domna) [inline pyproject.toml:55]: Could you also change this `pynxtools` occurrences and replace it with nyaml. It seems that I forgot to change this when I created the repo

File: pyproject.toml [hunk 2]
```diff
@@ -73,7 +73,7 @@ ignore = [
     "PLR1714", # consider-using-in
     "PLR5501", # else-if-used
 ]
-fixable = ["ALL"]
+lint.fixable = ["ALL"]
 exclude = ["pynxtools/definitions/*"]
```
Comment (domna) [inline pyproject.toml:77]: I think this exclude can also be removed

**Review by domna — APPROVED** [review]
It's fine but could you please check the pynxtools occurences in the pyproject toml? Maybe it is also in other settings.

File: pyproject.toml (hunk 2 above)
Comment (lukaspie) [reply pyproject.toml:77]: Sorry, you are right, I was too fast with the copy. Will change.

Comment (domna) [reply pyproject.toml:77]: Nothing to be sorry about. I think I actually forgot this when I created this file as this is not changed by your PR.

File: pyproject.toml (hunk 1 above)
Comment (domna) [reply pyproject.toml]: Yes, it looks all good now. Thanks!

**Pull request merged**